tensorflow/contrib/spectral/python/ops/shape_ops.py

```diff
@@ -0,0 +1,52 @@
+import math
```

Comment:
Unfortunately, the other packages are not that consistent, but I will try my best!